src/day-1-toy/fileio.py

```diff
@@ -1,12 +1,19 @@
 # Use open to open file "foo.txt" for reading
+f = open('foo.txt', 'r')
 
```

Comment:
This also works so you don't have to explicitly close the file
`with open('foo.txt') as textFile:
  text = textFile.read()
print(text)`